CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 Note: This is in reverse chronological order, so newer entries are added to the top.
 
-Swift 3.0
+Swift 3.1
```

Comment:
I thought Swift 3.0.1 will use the same `swift-3.0-branch`? The changes for SE-0045 are only in the `master` branch.